src/service/json-schema-service.js

```diff
@@ -4,14 +4,18 @@ import getSchema from '../controllers/http-api/request-schema/get-schema.js';
 import querySchema from '../controllers/http-api/request-schema/query-schema.js';
 import bidSuggestionSchema from '../controllers/http-api/request-schema/bid-suggestion-schema.js';
 import localStoreSchema from '../controllers/http-api/request-schema/local-store-schema.js';
+import { BID_SUGGESTION_OPTIONS } from '../constants/constants.js';
```

Comment:
this import can be moved to the schema file, there is no need to pass this as parameter since its a constant